.github/workflows/release-wheels.yml

```diff
@@ -38,19 +39,26 @@ jobs:
       - uses: actions/upload-artifact@v4
         with:
           path: dist/*.tar.gz
+          name: artifact-sdist
 
   upload_pypi:
     needs: [build_wheels, build_sdist]
     runs-on: ubuntu-latest
+    permissions:
+      contents: write  # IMPORTANT: mandatory for making GitHub Releases
```

Comment:
This privilege looks unused, FYI.

Reply:
I wasn’t entirely sure whether we’re planning to stick with the GitHub release workflow or move toward just pushing tags, happy to align either way.

Reply:
My typical workflow outside of aio-libs is a bit different, but also publishes to GH releases. I'd love to eventually migrate to that. But for now, whatever you're going to set up will be good.